Tools/WrapperGenerator/Templates/TypeScript/Partials/ResultName.mustache

```diff
@@ -0,0 +1,12 @@
+
```

Comment:
This partial looks to only be used in a single place, so it's wasteful to have it in a separate file.